pmm-app/src/pmm-dbaas/components/XtraDB/XtraDB.test.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { XtraDB } from './XtraDB';
+
+jest.mock('shared/components/helpers/notification-manager');
+
+describe('XtraDB::', () => {
+  it('renders correctly without clusters', () => {
+    const root = mount(
+      <XtraDB kubernetes={[]} />
+    );
+
+    expect(root.find('[data-qa="xtradb-add-cluster-button"]').find('button').length).toBe(2);
+    expect(root.contains('table')).toBeFalsy();
+  });
+});
```

Comment:
Please, add more tests here

Reply:
Added in other PR for PMM-5975, please check.